test/performance/benchmarks/load-test/main.go

```diff
@@ -190,10 +190,10 @@ func checkSLA(results *vegeta.Metrics, pacers []vegeta.Pacer, durations []time.D
 		expectedSum = expectedSum + pacers[i].Rate(time.Second)*durations[i].Seconds()
 	}
 	expectedRequests = uint64(expectedSum)
-	if results.Requests == expectedRequests {
-		log.Printf("SLA 4 passed. total requests is %d", results.Requests)
+	if results.Requests >= expectedRequests-(expectedRequests/1000) && results.Requests <= expectedRequests+(expectedRequests/1000) {
```

Comment:
Hm we should never have more requests that what we expect, right?

Reply:
You are right , I have submit a new commit

Reply:
You are right , I have submit a new commit